### ISSUE TITLE
As a user I want a feature to enable a more tolerant text sorting mode.

### ISSUE THREAD

**Issue by nilsschmidt1337** [nilsschmidt1337/ldparteditor#127]
A user once said...
>  Anybody experiencing that there is no sorting if the selection comprises a "0 // comment"?

LPE's sort algorithm does not destroy the parts structure. Comments between geometry are recognised as a start marker of the next section within the part structure.

Imagine a structure like this

```
0 // Front Faces
3 16 ...
3 16 ...
...
0 // Back Faces
3 16 ...
3 16 ...
```

Sorting will not mix front faces with back faces.

**I could make this behaviour optional later on (e.g. with a checkbox).**

**Comment by nilsschmidt1337** [comment]
Closed with commit 29df216.

**Comment by nilsschmidt1337** [comment]
The solution contained a bug.

**Comment by nilsschmidt1337** [comment]
Closed with commit f6ab62e.

**Comment by nilsschmidt1337** [comment]
The solution contained a bug.

**Comment by nilsschmidt1337** [comment]
Closed with commit 2054ac9.
